src/models.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+//------------------------------------------------------------------------------
+// Models for our locally stored data
+//------------------------------------------------------------------------------
+var Sequelize = require('sequelize');
+var Q = require('q');
+var AsyncLock = require('async-lock');
+
+
+var exports = (function() {
+    var _models = {};
+    var _lockQueue = [];
+    var _lock = new AsyncLock();
+
+    //--------------------------------------------------------------------------
+    // A lock that we can use to ensure that any writing to the database is 
+    // finished before another writer starts.
+    //
+    // A donePromise will be passed into the deferred, this donePromise should
+    // always be resolved when you are done with the database.
+    //--------------------------------------------------------------------------
+    function lock() {
+        var lockDeferred = Q.defer();
+        _lock.acquire("chesster", function() {
+            var unlock = Q.defer();
+            lockDeferred.resolve(unlock);
+            return unlock.promise;
+        });
+        return lockDeferred.promise;
+    }
+
+    //--------------------------------------------------------------------------
+    function defineModels(sequelize) {
+        module.exports.LichessRating = sequelize.define('LichessRating', {
```

Comment:
I assume you saw the comment about sequelize adding the two extra columns by now.  That's one way.

Also, these models will be different from the migrations if/when we add remove or change columns.  In that case, a new migration would be made and ONLY the change would be inserted into that migration.  At that point, this model definition would represent the current definition, and the migrations would represent the original definition followed by a series of changes that would modify the original definition into the current definition.